docs/styles/height.md

```diff
@@ -0,0 +1,37 @@
+# Height
+
+The `height` rule sets a widget's height. By default, it sets the height of the content area, but if `box-sizing` is set to `border-box` it sets the height of the border area.
+
+## Example
+
+=== "height.py"
+
+    ```python
+    --8<-- "docs/examples/styles/height.py"
+    ```
+
+=== "Output"
+
+    ```{.textual path="docs/examples/styles/height.py"}
+    ```
+
+## CSS
+
+```sass
+/* Explicit cell height */
+height: 10;
+
+/* Percentage height */
+height: 50%;
+
+/* Automatic height */
+width: auto
```

Comment:
Without saying what `auto` means this isn't very valuable. In fact, I've totally forgotten what it means and would love if these docs told me 😄

Reply:
Ah yeah. Will add.

Reply:
Actually, there is rather a lot to explain regarding dimensions, not just auto. I'll come back to this one.